pwiz_tools/Shared/CommonUtil/Collections/NaturalComparer.cs

```diff
@@ -42,4 +46,75 @@ public static int Compare(string x, string y)
             return StrCmpLogicalW(x, y);
         }
     }
+
+    // Similar to the filename comparer, except that decimals are understood
+    // e.g. for the filename comparer, "123.4056" comes after "123.456" because 4056 > 456 -
+    // it doesn't seem them as two parts of a decimal value
+    public class NaturalStringComparer : IComparer<string>
```

Comment:
Why can't this be static? Something to do with the Regex?

Reply:
Oh nevermind. Can't be IComparer if it's static., duh.

Reply:
Well, maybe not nevermind. The original code called Sort with NaturalComparer.Compare without needing to `new` the Comparer class. Why does the new code need `new`?

Reply:
You're probably right, could probably have a static. I'll try that.

Reply:
Yeah, we don't really need it to implement IComparer, may as well keep it simple. I've changed it, thanks.